src/index.js

```diff
@@ -1 +1,12 @@
+/* eslint-disable no-console */
 'use strict';
+
+require('dotenv/config');
+
+const { createServer } = require('./createServer');
+
+const PORT = process.env.PORT || 3005;
+
+createServer().listen(PORT, () => {
+  console.log('Server is running on localhost:5700');
```

Comment:
The console log message in the `createServer().listen` callback is incorrect. It should log the actual port the server is running on, which is stored in the `PORT` variable. Consider updating the message to use `PORT` instead of hardcoding '5700'.